src/test/java/no/ssb/subsetsservice/controller/SubsetsControllerV2Test.java

```diff
@@ -60,6 +64,41 @@ public JsonNode readJsonFile(File file) {
         return null;
     }
 
+    @BeforeAll
+    static void CreateTables() {
+        try {
+            Connection con = ConnectionPool.getInstance().getConnection();
+            LOG.debug("Attempting subsets table and index creation...");
+            PreparedStatement preparedStatement = con.prepareStatement(Scripts.CREATE_SERIES);
+            LOG.debug("Crete series table");
+            preparedStatement.executeUpdate();
+
+
+            PreparedStatement preparedStatement1 = con.prepareStatement(Scripts.SET_OWNER_SERIES);
+            LOG.debug("Set owner of series table");
+            preparedStatement1.executeUpdate();
+
+            PreparedStatement preparedStatement2 = con.prepareStatement(Scripts.CREATE_VERSIONS);
+            LOG.debug("create versions table");
+            preparedStatement2.executeUpdate();
+
+            PreparedStatement preparedStatement3 = con.prepareStatement(Scripts.SET_OWNER_VERSIONS);
+            LOG.debug("set owner of versions table");
+            preparedStatement3.executeUpdate();
+
+            PreparedStatement preparedStatement4 = con.prepareStatement(Scripts.CREATE_INDEX);
+            LOG.debug("create index");
+            preparedStatement4.executeUpdate();
+            LOG.debug("connection closed");
+
+            con.close();
```

Comment:
Connections should be closed in a finally block. But I prefer using [try-with-resources](https://www.baeldung.com/java-try-with-resources)

Reply:
done takk